Task50/Google/SeleniumEasy/DropdownDemoPage.cs

```diff
@@ -0,0 +1,40 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+
+namespace Task50Pages;
+
+internal class DropdownDemoPage : BaseClass
+{
+    const string URL = "https://demo.seleniumeasy.com/basic-select-dropdown-demo.html";
+    const string MULTISELECT_LOCATOR = "multi-select";
+    
+
+    public DropdownDemoPage(IWebDriver driver) : base(driver)
+    {
+        _driver.Url = URL;
+    }
+
+    public int SelectMultipleOptions()
+    {
+        SelectElement multiSelect = new SelectElement(GetElementById(MULTISELECT_LOCATOR));
+
+        if (multiSelect.IsMultiple)
+        {
+            multiSelect.SelectByIndex(0);
+            multiSelect.SelectByIndex(1);
+            multiSelect.SelectByIndex(2);
+        }
+
+        IList<IWebElement> options = new List<IWebElement>();
+
+        foreach (var option in multiSelect.Options)
```

Comment:
Instead of this look use this method to return selected options - https://www.selenium.dev/selenium/docs/api/dotnet/html/P_OpenQA_Selenium_Support_UI_SelectElement_AllSelectedOptions.htm